coreapi/sfdc/src/main/java/com/jalasoft/sfdc/ui/pages/FormBookBaseClassic.java

```diff
@@ -0,0 +1,33 @@
+package com.jalasoft.sfdc.ui.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class FormBookBaseClassic extends FormBoookPage {
+
+    @FindBy(xpath = "//*[@id=\"Name\"]")
+    WebElement viewN;
+    @FindBy(xpath = "//*[@id=\"Description\"]")
+    WebElement viewUName;
+//    @FindBy(xpath ="//*[@id=\"bottomButtonRow\"]/input[1]" )
+//    WebElement saveBtn;
+    public  FormBoookPage clickNewBook(){
+    return new FormBookBaseClassic();    }
```

Comment:
again no action to return the page, you  must perform some action with some webelement